labelindex/label_inheritance_index.go

```diff
@@ -66,7 +67,7 @@ type itemData struct {
 }
 
 // Get implements the Labels interface for itemData.  Combines the item's own labels with that
-// of its parents on the fly.
+// of its parent on the fly.
```

Comment:
that -> those (as there are multiple labels, in general)
parent -> parents (as an item can have multiple parents)